libs/documentation/src/lib/components/icons/demos/stacking/icons-stacking.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './icons-stacking.component.html',
+  styleUrls: ['./icons-stacking.component.scss']
```

Comment:
Can you delete the scss file since it's just an empty file? And also add in a selector - something like `icons-stacking-demo` or `sds-icons-stacking-demo` would be good.

Reply:
I've removed the scss file and updated the component.ts file to reflect that. I don't think we need a selector for this component. If added, it wouldn't be used as the demo framework does not rely on selectors to display components.